### PR TITLE
Add observations to SimulationTimeSeriesMatrix

### PR DESCRIPTION
**Pull request by jorgenherje**
Add observations to plot

---

Closes: #448

### CODE REVIEW

**Review by anders-kiaer — APPROVED** [review]
Nice 👍 Probably a separate issue: Should we (and if so how) inform the user if there is no summary observations for selected ensemble(s)?

**Comment by jorgenherje** [comment]
> Nice 👍 Probably a separate issue: Should we (and if so how) inform the user if there is no summary observations for selected ensemble(s)?

Good point. Perhaps we could provide it as a warning text to the `statusWriter`? If so, is it of interest to have it per selected vector or for the selected ensemble?

**Comment by anders-kiaer** [comment]
Most of the vectors in an ensemble with summary observations will still not have observations (i.e. there are only a few of the vectors in the set of vectors which potentially will have observations), so in order to avoid warning fatigue maybe try ensemble level?

**Comment by jorgenherje** [comment]
I've rewritten the query code to easier utilize data info and to show status when an ensemble does not have observations.

Made use of the new `combine`, where I see if the ensemble query gets any summary observations. If empty list from back-end is retrieved, the ensemble does not have observations.

**Review by HansKallekleiv — APPROVED** [review]
👍

File: frontend/src/modules/SimulationTimeSeriesMatrix/utils/subplotBuilder.ts [hunk 1]
```diff
@@ -466,8 +472,36 @@ export class SubplotBuilder {
         }
     }
 
-    addVectorObservations(): void {
-        throw new Error("Method not implemented.");
+    addVectorObservations(
+        vectorsObservationData: {
```
Comment (rubenthoms) [inline frontend/src/modules/SimulationTimeSeriesMatrix/utils/subplotBuilder.ts:476]: `vectorObservationData`?

**Review by rubenthoms — APPROVED** [review]
LGTM :+1: Two small comments.

**Pull request merged**